cmake/onnxruntime_python.cmake

```diff
@@ -71,9 +71,7 @@ onnxruntime_add_shared_library_module(onnxruntime_pybind11_state ${onnxruntime_p
 
 if(MSVC)
   target_compile_options(onnxruntime_pybind11_state PRIVATE "$<$<COMPILE_LANGUAGE:CUDA>:SHELL:--compiler-options /utf-8>" "$<$<NOT:$<COMPILE_LANGUAGE:CUDA>>:/utf-8>")
-  if(onnxruntime_ENABLE_TRAINING)
-    target_compile_options(onnxruntime_pybind11_state PRIVATE "/bigobj")
-  endif()
+  target_compile_options(onnxruntime_pybind11_state PRIVATE "/bigobj")
```

Comment:
We might have a source file that is too big and too complex. Ideally we should split the source file that was causing problems, instead of relying on this "/bigobj" flag.

Reply:
This started happening when added a small lora bindings which probably means it is not so much about individual obj file, but rather the combined size of them.

The fact that it compiled without training, but needed this flag with training bindings also may indicate it is about combined size.